### ISSUE TITLE
Tensorflow Savedmodel Export doesn't work

### ISSUE THREAD

**Issue by tr7zw** [googlecreativelab/teachablemachine-community#34]
When trying to download the model as a "Savedmodel", after converting it in the cloud, the browser tries to redirect to a 404 page and the downloaded zip just contains the labels.txt and an empty folder "model.savedmodel", but not the actual model. The keras export on the other hand now seems to work, but also tries to redirect to the 404.

**Comment by irealva** [comment]
Thanks for filing the bug! We're working on a fix.

**Comment by Joacim12** [comment]
Any news on this?

**Comment by irealva** [comment]
This should be fixed in latest release. Thanks for reporting!
